packages/graph-engine/src/nodes/series/power.ts

```diff
@@ -0,0 +1,91 @@
+import { INodeDefinition, ToInput, ToOutput } from '../../index.js';
+import { Node } from '../../programmatic/node.js';
+import { NumberSchema } from '../../schemas/index.js';
+import { arrayOf } from '../../schemas/utils.js';
+import { setToPrecision } from '../../utils/precision.js';
+
+type PowerValue = {
+	index: number;
+	value: number;
+};
+
+export default class NodeDefinition extends Node {
```

Comment:
This node feels a little weird, it expects to always generate a linearly incrementing power series, but it feels like it would be more expressive if I could chain it with say an alternating sequence or the output of a linear space